go.mod

```diff
@@ -81,3 +81,5 @@ retract (
 	// Published v1 too early (before migrating to go modules)
 	[v1.0.0, v1.0.1]
 )
+
+replace golang.org/x/tools => github.com/u-root/tools v0.1.9-0.20220120203125-f542f3b69e66
```

Comment:
this is a real bummer because anybody importing u-root now also needs to copy & paste this replace statement, but alas... it is what it is.